src/test/java/bad/robot/excel/workbook/PoiWorkbookTest.java

```diff
@@ -149,4 +154,16 @@ public void replaceThenLoadBlankCell() throws IOException {
         assertThat(adaptPoi(getCellForCoordinate(coordinate(A, 1), modified.workbook())), is(instanceOf(BlankCell.class)));
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldCountMergedCellsCorrectly() throws IOException, InvalidFormatException {
+        // Create a workbook with a single empty sheet that has merged cells.
+        XSSFWorkbook workbook = new XSSFWorkbook();
+        XSSFSheet sheet = workbook.createSheet("Sheet1");
+        int firstMergedCol = 1;
+        int lastMergedCol = 5;
+        sheet.addMergedRegion(new CellRangeAddress(0, 0, firstMergedCol, lastMergedCol));
+
+        // Compare the workbook to an empty workbook without merged cells.
+        assertThat(workbook, sameWorkbook(getWorkbookFromFactory("emptySheet.xlsx")));
```

Comment:
I changed this to use `getWorkbook` as they seem equivalent with the same result (i.e. a passing test)

Reply:
Yeah, that probably does not make a difference. I just wanted to stick as close as possible to my [original code](https://github.com/heremaps/oss-review-toolkit/blob/3fab2362a224f71639fb41b748b49ddd6255c481/reporter/src/funTest/kotlin/reporters/ExcelReporterTest.kt#L46) which uses `WorkbookFactory.create()` to rule out any unwanted side effects.